src/shared/serialize.c

```diff
@@ -403,6 +403,44 @@ int deserialize_pidref(FDSet *fds, const char *value, PidRef *ret) {
         return 0;
 }
 
+int deserialize_fd_set(const char *value, FDSet *fds) {
```

Comment:
you don't deserialize an fd set here, but only one fd

Reply:
well it's "from set"